File: Pet_Shop/Models/Servicos.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace Pet_Shop.Models
{
    public class Servicos
    {
        public int Cod { get; set; }


        [Required(ErrorMessage = "O Serviço deve ser informado")]
        [StringLength(50, MinimumLength = 3, ErrorMessage = "O nome deve ter de 3 à 50 caracteres")]
        public string Nome { get; set; }

        public int Valor { get; set; }
        public string Descricao { get; set; }

        public override string ToString()
        {
            return $"Cod: {Cod}, Nome: {Nome}, Valor: {Valor}, Descricao: {Descricao}";
        }
    }
}

File: Pet_Shop/wwwroot/js/models/Servicos.js

export default class Servicos {

    constructor(cod, nome, valor, descricao) {
        this.cod = cod;
        this.nome = nome;
        this.valor = valor;
        this.descricao = descricao;
    }

    get cod() {
        return this._cod;
    }
    set cod(novoCod) {
        this._cod = novoCod;
    }

    get nome() {
        return this._nome;
    }
    set nome(novoNome) {
        this._nome = novoNome;
    }

    get valor() {
        return this._valor;
    }
    set valor(novoValor) {
        this._valor = novoValor;
    }

    get descricao() {
        return this._descricao;
    }
    set descricao(novoDescricao) {
        this._descricao = novoDescricao;
    }

}